packages/replay/src/main.ts

```diff
@@ -711,7 +711,7 @@ async function updateMetadata({
   }
 }
 
-async function launchBrowser(browserName: BrowserName, args: string[] = []) {
+async function launchBrowser(browserName: BrowserName, detached: boolean, args: string[] = []) {
```

Comment:
It's probably not used by anyone but let's add detached to the end with a default value so we don't break the API.

Reply:
The `attach` parameter still needs to move after `args` so the API doesn't have a breaking change.